src/controls.actions.js

```diff
@@ -188,15 +188,19 @@
   /**
    * Wrap an action handler with saving/restoring object position on the transform.
    * this is the code that permits to objects to keep their position while transforming.
+   * @param {string} eventName
    * @param {Function} actionHandler the function to wrap
    * @return {Function} a function with an action handler signature
    */
-  function wrapWithFixedAnchor(actionHandler) {
+  function wrapWithFixedAnchor(eventName, actionHandler) {
```

Comment:
we can't change wrapWithFixedAnchor argument signature, or we would break the compatibility of fabric v4.

Reply:
we need an event wrapper, or something like that. I ll get some fixes done and then i ll try to help.